test/CMakeLists.txt

```diff
@@ -241,6 +241,34 @@ function(add_perf_test NAME)
 endfunction()
 
 set(IMPLICITGEMM_ARGS ${MIOPEN_TEST_FLOAT_ARG})
+# ./bin/MIOpenDriver conv -n 128 -c 1024 -H 14 -W 14 -k 2048 -y 1 -x 1 -p 0 -q 0 -u 2 -v 2 -l 1 -j 1 -m conv -g 1 -F 1 -t 1
+# MIOPEN_DEBUG_CONV_IMMED_FALLBACK=0
+if(MIOPEN_EMBED_DB)
+    set(MIOPEN_EMBED_TEST_ARG ${MIOPEN_TEST_FLOAT_ARG} --disable-validation --verbose)
```

Comment:
@JehandadKhan Could you please elaborate a bit why we need `--disable-validation` here?

Reply:
we don't, this is an oversight.

Reply:
Please remove it if you have time or let me know and I will do so.